invenio_assets/cli.py

```diff
@@ -38,6 +40,25 @@
 __all__ = ('collect', 'npm', )
 
 
+def amd_build_deprecation_warning():
+    """Add deprecation warning for AMD assets build system."""
+    import warnings
+    warnings.warn('The AMD build system is deprecated, please use Webpack '
+                  'with the webpack command.', DeprecationWarning)
```

Comment:
This should be a ``PendingDeprecationWarning``

Reply:
Updated with the changes. @lnielsen what about adding the link to the docs on how to migrate?